src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommandProviders.cs

```diff
@@ -42,7 +43,7 @@ private static IReadOnlyList<IVulnerabilityInformationProvider> CreateVulnerabil
             {
                 if (remoteProviders[i].SourceRepository != null)
                 {
-                    providers.Add(new VulnerabilityInformationProvider(remoteProviders[i].SourceRepository, NullLogger.Instance, isAuditSource: false));
+                    providers.Add(new VulnerabilityInformationProvider(remoteProviders[i].SourceRepository, NullLogger.Instance, isAuditSource: false, CancellationToken.None));
```

Comment:
I was going to comment that this means the request can't be cancelled, but this class (well, all public constructors) is marked as obsolete, so this line is never executed